### ISSUE TITLE
[REQ] API so we can update a user's group in the database, user maintenance

### ISSUE THREAD

**Issue by crashzk** [ianlucas/cs2-inventory-simulator#150]
I'm looking at a CSSharp plugin whose function would be to update user groups in the database so that it can free up resources for them in the inventory simulator.

It would be interesting to have a user maintenance API, with which we could do:
- Delete, create and even associate a user with a group.

**_NOTE: I'm just unsure about whether a user should be created or not if the player doesn't exist in the database._**

The plugin, as we understand it, will perform some checks before assigning the group to the player:
- Check if the user has VIP status on the CS2 server,
- Check if the user already exists in the Inventory Simulator database,
- And only if both checks are positive, assign the role.

If the user is not in the website database, the plugin does nothing.

If the user is not a VIP on the CS2 servers, the plugin does nothing.

In other words, both checks must be true.

And if the user is registered on the website, is a VIP but the VIP expires, we will remove him from the group via the API as well.

**Comment by crashzk** [comment]
@Ferks-FK who is helping me with the plugin and the request to create the API, I will leave it with ping just so we can check the points and proceed with this implementation.

**Comment by crashzk** [comment]
Hey, could this be done now? What would the development roadmap for the project look like?
